safe/processors/post_processor_functions.py

```diff
@@ -56,6 +58,82 @@ def size(size_calculator, geometry):
     return feature_size
 
 
+def calculate_distance(
+        distance_calculator,
+        place_geometry,
+        latitude,
+        longitude):
+    """Simple postprocessor where we compute the distance between two points.
+
+    :param distance_calculator: The size calculator.
+    :type distance_calculator: safe.gis.vector.tools.SizeCalculator
+
+    :param latitude: The latitude to use.
+    :type latitude: float
+
+    :param longitude: The longitude to use.
+    :type longitude: float
+
+    :param place_geometry: Geometry of place.
+    :type place_geometry: QgsGeometry
+
+    :return: distance
+    :rtype: float
+    """
+    epicenter = QgsPoint(longitude, latitude)
+    place_point = place_geometry.asPoint()
+    distance = distance_calculator.measure_distance(epicenter, place_point)
+    return distance
+
+
+def calculate_bearing(place_geometry, latitude, longitude):
+    """Simple postprocessor where we compute the bearing angle between two
+    points.
+
+    :param place_geometry: Geometry of place.
+    :type place_geometry: QgsGeometry
+
+    :param latitude: The latitude to use.
+    :type latitude: float
+
+    :param longitude: The longitude to use.
+    :type longitude: float
+
+    :return: Bearing angle
+    :rtype: float
+    """
+    epicenter = QgsPoint(longitude, latitude)
+    place_point = place_geometry.asPoint()
+    bearing = place_point.azimuth(epicenter)
+    return bearing
+
+
+def calculate_cardinality(angle):
+    """Simple postprocessor where we compute the cardinality of an angle.
+
+    :param angle: Bearing angle.
+    :type angle: float
+
+    :return: Cardinality text.
+    :rtype: str
+    """
+    # this method could still be improved later, since the acquisition interval
+    # is a bit strange, i.e the input angle of 22.499° will return `N` even
+    # though 22.5° is the direction for `NNE`
+    direction_list = [
+        'N', 'NNE', 'NE', 'ENE', 'E', 'ESE', 'SE',
```

Comment:
Can you add the `tr` function? It French, I need to replace `W` by `O` : https://upload.wikimedia.org/wikipedia/commons/thumb/2/26/Compass_Rose_French_North.svg/220px-Compass_Rose_French_North.svg.png

Reply:
maybe for a context in translations, it's ok to do 
`tr('N,NNE,NE,ENE,E, ESE,SE,all_of_them').split(',')` so we have only one string to translate in transifex and the translator can understand what it means and then the code will split each of them.

Reply:
OK, I will add it.